pkg/utils/metrics.go

```diff
@@ -73,21 +74,16 @@ func HandleUserMetricsPreference(app *application.Avalanche) error {
 	} else {
 		ux.Logger.PrintToUser("Thank you for opting in Avalanche CLI usage metrics collection")
 	}
-	saveMetricsConfig(app, yes)
+	saveMetricsConfig(yes)
 	return nil
 }
 
-func userIsOptedIn(app *application.Avalanche) bool {
-	// if config file is not found or unable to be read, will return false (user is not opted in)
-	config, err := app.LoadConfig()
-	if err != nil {
-		return false
-	}
-	return config.MetricsEnabled
+func userIsOptedIn() bool {
+	return viper.GetBool(constants.MetricsEnabled)
```

Comment:
call a function on app, maybe app.GetConfigValue, that returns an interface to be converted afterwards

Reply:
not sure how you want to implement this